lighthouse-cli/package.json

```diff
@@ -1,6 +1,7 @@
 {
   "main": "index.js",
   "scripts": {
+    "prepublish": "npm run build",
```

Comment:
this is basically the purpose of `prepublish`, but like with `install` below, it feels weird to me to build everything, test it for release, then run `npm publish` and have everything rebuilt (albeit theoretically to the same thing just tested) before being pushed. We can try it out and see if there are issues, I guess.